snmp/datadog_checks/snmp/data/default_profiles/cisco-firepower.yaml

```diff
@@ -0,0 +1,133 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+  # This profile does not import cisco.yaml on purpose
+sysobjectid:
+  - 1.3.6.1.4.1.9.1.2404           # Cisco FPR 2110td
+  - 1.3.6.1.4.1.9.1.2405           # Cisco FPR 2120td
+  - 1.3.6.1.4.1.9.1.2406           # Cisco FPR 2130td
+  - 1.3.6.1.4.1.9.1.2407           # Cisco FPR 2140td
+  - 1.3.6.1.4.1.9.12.3.1.3.1788    # Cisco FPR 4120
```

Comment:
I'm perplexed by the use of this sysOID and the CISCO-ENTITY-VENDORTYPE-OID-MIB instead of the CISCO-PRODUCTS-MIB
for FPR 4120 we have sysOIDs `1.3.6.1.4.1.9.1.2294` for FPR 4120 K9 and `1.3.6.1.4.1.9.1.2314` for FPR 4120 SM24 from the CISCO-PRODUCTS-MIB